packages/botonic-core/src/core-bot.js

```diff
@@ -15,10 +15,10 @@ export class CoreBot {
     appId,
     defaultTyping,
     defaultDelay,
-    defaultRoutes
+    defaultRoutes,
   }) {
     this.renderer = renderer
-    this.plugins = loadPlugins(plugins)
+    this.plugins = plugins ? loadPlugins(plugins) : []
```

Comment:
Maybe this logic can be done inside `loadPlugins` in file `src/plugins` because we are checking there if `plugins` is not defined. We need to return only `[]` instead of `null`

Reply:
yes, i'm agree, i have refactored it as you suggest 👍